src/scenes/AcceleratorRouter/Scoring/EditView.tsx

```diff
@@ -41,16 +43,20 @@ const ScorecardEditView = () => {
   }, [history, projectId]);
 
   const onCancel = useCallback(() => {
-    goToScorecardView();
-  }, [goToScorecardView]);
+    if (hasData === false) {
```

Comment:
should this be `!hasData` to handle `hasData === undefined`?

Reply:
If it is undefined, just means that the calculation hasn't happened yet. In this edit use-case, it won't be undefined.
Mostly helps in the view use-case.